hydra-node/test/Hydra/Chain/Direct/Contract/Init.hs

```diff
@@ -1,3 +1,5 @@
+{-# OPTIONS_GHC -Wno-unused-imports #-}
```

Comment:
suggestion: Remove, I have added this inadvertendly as a Wip

Reply:
Whoops.